services/app-api/forms/wp.json

```diff
@@ -1127,6 +1127,196 @@
+            "addEditModalAddTitle": "Add funding source and projected expenditures",
+            "addEditModalEditTitle": "Edit funding source and projected expenditures",
+            "deleteEntityButtonAltText": "Delete other target population",
+            "deleteModalTitle": "Delete funding source for this report? ",
+            "deleteModalConfirmButtonText": "Yes, delete funding source",
+            "deleteModalWarning": "You will lose all information entered for this funding source. Funding source will not be deleted from archived Work Plans and Semi-Annual Reports, but it will not be available in future Work Plans and Semi-Annual Reports. <br><br>Are you sure you want to proceed?",
+            "tableHeader": "Initiative name <br/> Work Plan topic",
+            "addEditModalHint": "Provide projected quarterly expenditures, by funding source, for this initiative. Actual quarterly expenditures will be reported in the recipient’s Semi-Annual Report (SAR).",
+            "emptyDashboardText": "Empty",
+            "dashboardTitle": "Funding Sources"
+          },
+          "modalForm": {
+            "id": "tb-modal",
+            "fields": [
+              {
+                "id": "initiative_wpTopic",
+                "type": "radio",
+                "validation": "radio",
+                "props": {
+                  "label": "Funding source:",
+                  "hint": "Enter a dollar amount using a $ or a percentage with %. Enter 0 for quarters with no projected expenditures. Enter N/A for quarters you do not expect to report.",
+                  "choices": [
+                    {
+                      "id": "2VLpZ9A92OivbZhKvY8pE4hB65c",
+                      "label": "State-equivalent funds"
+                    },
+                    {
+                      "id": "2VLpZ9A92OivbZhKvY8pE4",
+                      "label": "Capacity-building funds"
+                    },
+                    {
+                      "id": "2VLpZCNtbcjRPq3evd1NI6",
+                      "label": "Administrative cooperative agreement funding"
+                    },
+                    {
+                      "id": "2VLpZDJ9qaKKOk78ztBdiB",
+                      "label": "MFP funding for qualified HCBS"
+                    },
+                    {
+                      "id": "2VLpZCRWieGr1Z49QX5Aqc",
+                      "label": "MFP funding for demonstrated services"
+                    },
+                    {
+                      "id": "2VLq8ASzZNfxbu520if449",
+                      "label": "MFP funding for supplemental services"
+                    },
+                    {
+                      "id": "2VLq8ASzZNfxbu520if449",
+                      "label": "Other, specify",
+                      "children": [
+                        {
+                          "id": "initiative_wp_otherTopic",
+                          "type": "textarea",
+                          "validation": {
```

Comment:
I think this text area needs to be the one that can't be expanded (again, going off the Figma)